bigquery/deploy/README.md

```diff
@@ -46,7 +46,7 @@
   * Create cluster
     * Go to https://console.cloud.google.com/kubernetes/list and click `Create Cluster`
     * Change name to `elasticsearch-cluster`
-    * Change `Machine type` to `4 vCPUs`. (Otherwise will get Insufficient CPU error.)
+    * Change `Machine type` to `n1-highmem-4` (4vCPUs, 26GB memory)
```

Comment:
```suggestion
    * Change `Machine type` to `4 vCPUs`. (Otherwise will get Insufficient CPU error.)
    If you need more memory for Elasticsearch, you may need `n1-highmem-4`. [Elasticsearch recommends](https://www.elastic.co/guide/en/elasticsearch/reference/current/heap-size.html) the VM has at least
    twice the memory you are using for Elasticsearch.
```